core/deployment/pom.xml

```diff
@@ -142,6 +145,24 @@
                             <goal>enforce</goal>
                         </goals>
                     </execution>
+                    <execution>
+                        <id>enforce-quarkus-core-deployment</id>
+                        <configuration>
+                            <rules>
+                                <bannedDependencies>
+                                    <excludes>
+                                        <!-- findbugs is not required at runtime -->
+                                        <exclude>com.google.code.findbugs:jsr305</exclude>
```

Comment:
This dependency is managed in the quarkus-bom. It's a runtime dependency of quarkus-grpc.

Reply:
That's fine for grpc and I do not think it contradicts banning it here in quarkus-core-deployment. I mean it would be nice if findbugs does not soak to every single deployment module that depends on quarkus-core-deployment. Most, if not all deployment modules depend on quarkus-core-deployment.